### ISSUE TITLE
Hook up color reduction to imageclean step

### ISSUE THREAD

**Issue by mejackreed** [sul-cidr/histonets#54]

**Comment by csbailey5t** [comment]
We need to add a selector for either kmeans or linear as the method for posterize. I'll create a new issue for this.

**Comment by csbailey5t** [comment]
Before we can close this, we also need to make sure the option `-m` is passed through for posterize to select either kmeans or linear.
